packages/react-storage/src/components/StorageBrowser/context/actions/listLocationsAction.ts

```diff
@@ -8,6 +11,8 @@ import {
   Permission,
 } from '../types';
 
+const PAGE_SIZE = 1000;
```

Comment:
Extraneous, should be removed

Reply:
Discussed async, we need a default page size to recursively call the function since `pageSize` is an optional param. Updated to set default fallback pageSize to `100`